apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx

```diff
@@ -40,14 +41,27 @@ export function ValidatorsCard() {
         return getAllStakeSui(delegatedStake);
     }, [delegatedStake]);
 
+    const { data: inActiveStakingPoolID } = useGetInactiveValidators(
+        system?.inactivePoolsId
+    );
+
     const delegations = useMemo(() => {
         return delegatedStake?.flatMap((delegation) => {
             return delegation.stakes.map((d) => ({
                 ...d,
+                // flag any inactive validator for the stakeSui object
+                inactiveValidator: inActiveStakingPoolID?.includes(
+                    delegation.stakingPool
+                ),
                 validatorAddress: delegation.validatorAddress,
             }));
         });
-    }, [delegatedStake]);
+    }, [delegatedStake, inActiveStakingPoolID]);
+
+    // Check if there are any inactive validators
+    const inActiveStakes = delegations?.some(
```

Comment:
nit: it would be nice to be consistent with the capitalization here, `inActiveStakes` reads different than `inactiveValidator` and I'm unsure if `inActiveStakes` is referring to something that's inactive